frontend/lib/widgets/sidebar.dart

```diff
@@ -1,80 +1,68 @@
 import 'package:flutter/material.dart';
-import 'package:frontend/pages/loginPage.dart';
 
-class SideBar extends StatelessWidget {
-
-  void logout(context) { // TODO: Missing Actual Functionality 
-    Navigator.push(
-      context,
-      MaterialPageRoute(builder: (context) => LoginPage()),
-    );
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.only(left: 15),
-      child: ListView(
-        children: <Widget>[
-          SizedBox(height: 15),
-          FlatButton(
-            onPressed: () {
-              /*...*/
-            },
-            child: SizedBox(
-              width: double.infinity,
-                          child: Text(
-                "Home",
-                 textAlign: TextAlign.left,
+Widget SideBar(Function onPresss, Function logout) {
```

Comment:
Same as above

Reply:
Also we can use a class but I don't think it would affect much tbh.